features/environment.py

```diff
@@ -312,10 +340,7 @@ def before_scenario(context: Context, scenario: Scenario):
         releases = releases.intersection(context.config.filter_series)
```

Comment:
We have a problem here related to the `series.all` tag. When used alone, as is the case for aws pro, the `filter_series` ends up as `{ "all" }`. This means that this intersection here will return empty and will not create any of the images required for testing.